### PR TITLE
Rename GitHub Action file

### PR DESCRIPTION
**Pull request by ArangoGutierrez**

### CODE REVIEW

File: .github/workflows/image.yaml
Comment (elezar) [inline .github/workflows/image.yaml:1]: Why are we calling this images.yaml? We don't generate any images?

Comment (ArangoGutierrez) [reply .github/workflows/image.yaml:1]: Should we then edit the name to be packages, so is in line with the file name ?

Comment (ArangoGutierrez) [reply .github/workflows/image.yaml:1]: how about now?

**Review by elezar — APPROVED** [review]
Looks good.

Once this is merged, please create a PR to nvidia-container-toolkit that updates the git submodule.

Done: https://github.com/NVIDIA/nvidia-container-toolkit/pull/387

**Pull request merged**